mtags/src/main/scala-2/scala/meta/internal/pc/MetalsGlobal.scala

```diff
@@ -183,26 +202,31 @@ class MetalsGlobal(
       }
     }
 
+    def visitMember(sym: Symbol) = {
+      if (isRelevantWorkspaceSymbol(sym))
+        visit {
+          if (isInStringInterpolation)
+            new WorkspaceInterpolationMember(
+              sym,
+              Nil,
+              edit => s"{$edit}",
+              None
+            )
+          else
+            new WorkspaceMember(sym)
+        }
+      else false
+    }
+
+    // TODO the same in autoimports TEST
```

Comment:
Do we check auto imports? Is the TODO still relevant?

Reply:
I added tests, this seems to be already handled